models/categories.models.js

```diff
@@ -15,3 +15,19 @@ exports.fetchEndpoints = () => {
       return parsedEndpoints;
     });
 };
+
+exports.fetchReviewId = (id) => {
+  if (isNaN(id)) {
+    return Promise.reject({ status: 400, msg: "Bad request!" });
+  }
+
+  const queryStr = `SELECT * FROM reviews WHERE review_id = $1`;
+
+  return db.query(queryStr, [id]).then((result) => {
+    if (result.rows.length === 0) {
+      return Promise.reject({ status: 500, msg: "Server Error..." });
```

Comment:
Continuing on from my comment in the tests if `result.rows.length === 0` is true then why is this? It's not a server error. It's just that the review is "not found". 😄  We should send the appropriate error code/message back at this point.